tools/submission/submission-checker.py

```diff
@@ -1206,13 +1234,12 @@ def check_results_dir(config, filter_submitter,  skip_compliance, csv, debug=Fal
       "host_processors_per_node", "host_processor_core_count",
       "accelerator_model_name", "accelerators_per_node", "Location",
       "framework", "operating_system", "notes", "compilance", "errors",
-      "version", "inferred", "has_power", "Units"
+      "version", "inferred", "has_power", "Units", "network_mode"
```

Comment:
remove the field - add open/closed/network closed? maybe add to submission checker PR.

Reply:
done